src/midi/midi_fluidsynth.cpp

```diff
@@ -386,17 +526,6 @@ static void setup_chorus(fluid_synth_t* synth, const std_fs::path& sf_path)
 	                                  FxGroup,
 	                                  static_cast<int>(chorus_mod_wave));
 
-// deprecated API calls prior to 2.2
-#else
-	fluid_synth_set_chorus_on(synth, chorus_enabled);
-	fluid_synth_set_chorus(synth,
-	                       chorus_voice_count,
-	                       chorus_level,
-	                       chorus_speed,
-	                       chorus_depth,
-	                       chorus_mod_wave);
-#endif
-
```

Comment:
It's not the end of the world, but it would be nicer to do these all version check removals in a separate commit (and good job removing them 💯 ).

Reply:
I was thinking "but then I have to implement the symbols, just to remove them".

But duh, I can just remove them before introducing the dynamic loading system.

Reply:
> But duh, I can just remove them before introducing the dynamic loading system.

Yeah, that's exactly what I thought, let's do that 😄